postgresql.go

```diff
@@ -16,6 +16,10 @@ type PostgreSQL struct {
 	// which requires SUPERUSER privileges.
 	UseAlterConstraint bool
 
+	// DontResetSequences prevents the reset of the databases
+	// sequences after load fixtures time
+	DontResetSequences bool
```

Comment:
```suggestion
	SkipResetSequences bool
```

Reply:
I choose this negative logic so that the default value keep a compatible backward behaviour. But in the end its your call.